src/windows/main_window.py

```diff
@@ -1311,6 +1323,25 @@ def actionNextMarker_trigger(self, event):
             get_app().window.refreshFrameSignal.emit()
             get_app().window.propertyTableView.select_frame(frame_to_seek)
 
+            # Center the timeline on the marker
+            self.timeline.centerOnTime(closest_position)
+            
+    def actionCenterOnPlayhead_trigger(self, event):
+        log.info('actionCenterOnPlayhead_trigger')
```

Comment:
These are basically debug `printf()`s that shouldn't even be in the _existing_ code, it's fine for testing but I'd really rather not commit any new ones.

Reply:
Right.  Being unfamiliar with the code, I was mimicking what I saw elsewhere, but agree there's no point in keeping them in.

Reply:
\*nod\* That's what gets us. The code is fine in many ways, and passes the pragmatist's litmus test of, "well, it _works_." (And, really, that **is** the ultimate test of quality.) But it also contains a bunch of antipatterns that just keep getting reinforced by (bad) example.

(The bare `except:` clauses all over the Python code fall into that same category. Those aren't just bad, they're **#EEEVIL**. But there's still plenty of them in the code.)

Reply:
I believe I've removed all of my debug statements.  I didn't go after any others, though.  Even though I wanted to.